src/helpers/url.js

```diff
@@ -40,7 +40,7 @@ const getCallbackBaseUrl = () => {
 const getWopiSrc = (fileId) => {
 	// WOPISrc - URL that loolwsd will access (ie. pointing to ownCloud)
 	// index.php is forced here to avoid different wopi srcs for the same document
-	const wopiurl = getCallbackBaseUrl() + '/index.php/apps/richdocuments/wopi/files/' + fileId
+	const wopiurl = getCallbackBaseUrl() + 'index.php/apps/richdocuments/wopi/files/' + fileId
```

Comment:
Wouldn't it be better to ensure that `getCallbackBaseUrl()` returns URL without trailing `/`? I remember similar problems back in the day when the Collabora Online URL was misbehaving depending on whether the admins added trailing `/` or not, and IIRC it ended up being always removed (but I could be wrong).

If that's the case, perhaps not adding the trailing `/` in `getCallbackBaseUrl() ` would be consistent (but it also has to be ensured that `wopi_callback_url` is stored without a trailing `/`):
`return callbackUrl || window.location.protocol + '//' + window.location.host + getRootUrl() + '/'`
https://github.com/nextcloud/richdocuments/blob/b784eeb24dd1dfa32fdade11bdc814b1d312229e/src/helpers/url.js#L37

@juliushaertl Can you please share your thoughts?

Reply:
Yeah I think it would be more consistent if we trim any trailing slash from the base. There are other places where this is probably wise to do

Reply:
changed base then